### PR TITLE
Cities

### PR DESCRIPTION
**Pull request by willrstewart**
Here are some new things about my favorite city

### CODE REVIEW

File: PRONOUNCED.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+asdfPRONOUNCED
```
Comment (iangrey1) [inline PRONOUNCED.txt]: is this spelled correctly?

File: THAMES.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+asdfTHAMES
```
Comment (iangrey1) [inline THAMES.txt]: THAMES like THOMAS not THAMES like THE

File: PRONOUNCED.txt (hunk 1 above)
Comment (willrstewart) [reply PRONOUNCED.txt]: OOps

File: THAMES.txt (hunk 2 above)
Comment (willrstewart) [reply THAMES.txt]: Ok

Comment (iangrey1) [reply THAMES.txt]: this looks correct

File: PRONOUNCED.txt (hunk 1 above)
Comment (iangrey1) [reply PRONOUNCED.txt]: try PRODUCE'D

**Pull request merged**